# Patch needed for cdo version 2.0.6. Should be fixed in next version.
# See: https://code.mpimet.mpg.de/boards/2/topics/13186?r=13240#message-13240
# 
--- src/cdo_fft.cc.orig	2022-07-11 14:51:10.000000000 +0200
+++ src/cdo_fft.cc	2022-09-13 15:56:27.000000000 +0200
@@ -1,6 +1,7 @@
 // This source code is copied from PINGO version 1.5
 
 #include <cmath>
+#include <algorithm>
 
 namespace cdo
 {
